docker-compose.full.rules.yaml

```
@@ -1,694 +1,496 @@
-      - RULE_NAME=091
-      - APM_SERVICE_NAME=rule-091
-      - CONFIGURATION_DATABASE_USER=postgres
-      - CONFIGURATION_DATABASE_PASSWORD=unused
-      - EVENT_HISTORY_DATABASE_USER=postgres
-      - EVENT_HISTORY_DATABASE_PASSWORD=unused
-      - RAW_HISTORY_DATABASE_USER=postgres
-      - RAW_HISTORY_DATABASE_PASSWORD=unused
-    restart: always
-    depends_on:
-      - valkey
+# For local testing
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)